src/platform_impl/ios/view.rs

```diff
@@ -291,12 +412,28 @@ impl WinitView {
         }
     }
 
+    pub(crate) fn recognize_pan_gesture(&self, should_recognize: bool) {
+        if should_recognize {
+            if self.ivars().pan_gesture_recognizer.borrow().is_none() {
+                let pan: Id<UIPanGestureRecognizer> = unsafe {
+                    msg_send_id![UIPanGestureRecognizer::alloc(), initWithTarget: self, action: sel!(panGesture:)]
+                };
```

Comment:
You've documented the event as a "two-finger pan gesture", maybe I'm misunderstanding, but would it make sense to set `minimumNumberOfTouches` and `maximumNumberOfTouches` to 2?

Reply:
I agree, for the swipe gesture I needed to set min/max number of touches to disambiguate swiping from panning as well. I'll make it configurable on the recognize_pan_gesture function.